airflow-comparison.md

```diff
@@ -0,0 +1,47 @@
+## Lineage
+
+Data lineage broadly refers to a collection of tools, practices and approaches which allow the tracking of the flow of data within a data pipeline. Data lineage tools maintain a record of data within its lifecycle, including its source and any transformations applied to it. The main purpose of data lineage is to provide a platform for ensuring data quality, as well as a way of troubleshooting any errors which arise within the data.
```

Comment:
> The main purpose of data lineage is to provide a platform for ensuring data quality, as well as a way of troubleshooting any errors which arise within the data.

I would add data flow & movement traceability from source to destination, with a focus on the transformations made. Data lineage reflects the data transformation journey.